features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/pages/cdmf.page.user.view/view.js

```diff
@@ -17,6 +17,16 @@
  */
 
 function onRequest(context) {
+    context.handlebars.registerHelper('equal', function (lvalue, rvalue, options) {
+        if (arguments.length < 3)
+            throw new Error("Handlebars Helper equal needs 2 parameters");
+        if( lvalue!=rvalue ) {
```

Comment:
Acknowledged and fixed with commit https://github.com/wso2/carbon-device-mgt/commit/e537e2d841ca3c7946cd34e9da96f261e0ac94e1